molo/core/models.py

```diff
@@ -274,7 +274,8 @@ class ArticlePage(CommentedPageMixin, Page, TagSearchable):
         on_delete=models.SET_NULL,
         related_name='+'
     )
-
+    metadata_tags = models.TextField(null=True, blank=True,
```

Comment:
This should be of type `ClusterTaggableManager(through=ArticlePageTag, blank=True)` similar to the `tags` field